### DISCUSSION TITLE
Installation and Tutorials for ROS Noetic or ROS2

### DISCUSSION THREAD

**Discussion by 22222140** [lagadic/visp#1196] [Q&A]
Hello,

I started work on ROS-based tracking UAV, and UGV robots using ViSP

But, I just can find it available only for ROS Melodic. 
Can I get any tutorial and code for implementation on ROS Noetic or ROS2

**Comment by fspindle** [comment]
Look at https://github.com/lagadic/vision_visp and https://github.com/lagadic/visp_ros from which you can get inspiration for ros1 and ros2
